detailed_hashimae_info.py

```
@@ -0,0 +1,129 @@
+                else:
+                    print("❌ 橋前が見つかりませんでした")
+                    
+                # Show structured data section
+                if "構造化データ" in result:
+                    structured_section = result.split("--- 構造化データ ---")[1]
+                    try:
+                        structured_data = json.loads(structured_section)
+                        for field in structured_data.get('fields', []):
+                            if field.get('name') == '橋前':
+                                print(f"📊 構造化面積: {field.get('area')}")
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)